README.md

```diff
@@ -209,127 +244,149 @@ See [LoopBack types](http://loopback.io/doc/en/lb3/LoopBack-types.html) for de
 
 _NOTE_ as of v3.0.0 of MySQL Connector, the following flags were introduced:
 
-- `treatCHAR1AsString`
-  default `false` - treats CHAR(1) as a String instead of a Boolean
-- `treatBIT1AsBit`
-  default `true` - treats BIT(1) as a Boolean instead of a Binary
-- `treatTINYINT1AsTinyInt`
-  default `true` - treats TINYINT(1) as a Boolean instead of a Number
-
-## Using the datatype field/column option with MySQL
-
-Use the `mysql` model property to specify additional MySQL-specific properties for a LoopBack model.
+- `treatCHAR1AsString` default `false` - treats CHAR(1) as a String instead of a
+  Boolean
+- `treatBIT1AsBit` default `true` - treats BIT(1) as a Boolean instead of a
+  Binary
+- `treatTINYINT1AsTinyInt` default `true` - treats TINYINT(1) as a Boolean
+  instead of a Number
+
+## Data mapping properties
+
+### Table/Column Names
+
+Besides the basic LoopBack types, as we introduced above, you can also specify
+additional MySQL-specific properties for a LoopBack model. It would be mapped to
+the database.
+
+Use the `mysql.<property>` in the model definition or the property definition to
+configure the table/column definition.
+
+For example, the following settings would allow you to have custom table name
+(`Custom_User`) and column name (`custom_id` and `custom_name`). Such mapping is
+useful when you'd like to have different names from the model:
+
+{% include code-caption.html content="user.model.ts" %}
+
+```ts
+@model({
+  settings: { mysql: { schema: 'testdb', table: 'Custom_User'} },
+})
+export class User extends Entity {
+  @property({
+    type: 'number',
+    required: true,
+    id: true,
+    mysql: {
+      columnName: 'custom_id',
+    },
+  })
+  id: number;
 
-For example:
+  @property({
+    type: 'string',
+    mysql: {
+      columnName: 'custom_name',
+    },
+  })
+  name?: string;
+```
 
-{% include code-caption.html content="/common/models/model.json" %}
+<details><summary markdown="span"><strong>For LoopBack 3 users</strong></summary>
 
 ```javascript
-"locationId":{
-    "type":"String",
-    "required":true,
-    "length":20,
-    "mysql":
-    {
-        "columnName":"LOCATION_ID",
-        "dataType":"VARCHAR",
-        "dataLength":20,
-        "nullable":"N"
+{
+  "name": "User",
+  "options": {
+    "mysql": {
+      "schema": "testdb",
+      "table": "Custom_User"
     }
+  },
+  "properties": {
+    "id": {
+      "type": "Number",
+      "required": true,
+      "mysql": {
+        "columnName": "custom_id",
+      }
+    },
+    "name": {
+      "type": "String",
+      "mysql": {
+        "columnName": "custom_name",
+      }
+    },
+  }
 }
 ```
 
-You can also use the dataType column/property attribute to specify what MySQL column type to use for many loopback-datasource-juggler types. 
-The following type-dataType combinations are supported:
+</details>
+
+### Digital Types
```

Comment:
do you mean data types?

Reply:
I think Numeric is the correct word, fixed.